src/drivers/distance_sensor/broadcom/afbrs50/AFBRS50.cpp

```diff
@@ -294,6 +311,25 @@ static int stop()
 	return PX4_OK;
 }
 
+static int test(const uint8_t rotation)
```

Comment:
These changes all look good to me, I just might be missing why this method needs to be static.